faqs.html

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: FAQs
+---
+<div class="mk-masthead__content--sub">
+    <h1 class="mk-masthead__content--sub__title">Frequently Asked Questions</h1>
+    <p>Here are some answers to common questions, discover more about Metal<sup>3</sup></p>
+    </div>
+<section class="mk-faqs mk-main__section">
+    <div data-aria-faq data-multi data-transition data-default="none" id="accGen">
+        {% for faq in site.faqs %}
+        <h3 class="mk-heading--md" data-aria-faq-heading>
+            {{ faq.question }}
+        </h3>
+        <div data-aria-faq-panel>
+          {{ faq.content }}
+      </div>
+        {% endfor %}
+    </div>
+    </section>
+
+
+
+    <script>
+
+        ! function(t, e, a) {
```

Comment:
What is this and why is it needed?
My gut feeling says "we just want a simple FAQ, get rid of the script!"

Reply:
The script was used because most of the accordion styling was done in the script, i needed to use it so the styling would be consistent

Reply:
So will you remove the script or will it stay @SafeEHA ? The answer wasn't clear to me.

Reply:
or did you mean "you wouldn't implement them now but it was already implemented in the past and you are just carrying it over" ?